flutter-studio/src/io/flutter/project/FlutterProjectModel.java

```diff
@@ -150,4 +132,26 @@ protected void handleFinished() {
     }
     new FlutterProjectCreator(this).createProject();
   }
+
+  @NotNull
+  private static String getInitialDomain() {
+    String domain = PropertiesComponent.getInstance().getValue(PROPERTIES_DOMAIN_KEY);
```

Comment:
I think this might also work?

    return PropertiesComponent.getInstance().getValue(PROPERTIES_DOMAIN_KEY, DEFAULT_DOMAIN);

Reply:
I think you're right and I'll keep that in mind if I ever edit this code. It's been like this for a year or so, and is just now getting properly reformatted and reorganized.